.github/workflows/share-new-blog-posts.yml

```diff
@@ -0,0 +1,71 @@
+---
+name: Share new blog posts
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "application/src/data/post/**"
+  workflow_dispatch: {}
```

Comment:
only on workflow_call, must be call on main-ci success, or workflow_dispatch must force to specify blog post to share